src/iperf_locale.c

```diff
@@ -196,6 +196,9 @@ const char usage_longstr[] = "Usage: iperf3 [-s|-c host] [options]\n"
                            "  --dscp N or --dscp val    set the IP dscp value, either 0-63 or symbolic.\n"
                            "                            Numeric values can be specified in decimal,\n"
                            "                            octal and hex (see --tos above).\n"
+#if defined(HAVE_SO_PRIORITY)
+                           "      --sock-prio N         set the socket priority (only supported on Linux)\n"
```

Comment:
The `(only supported on Linux)` is redundant (and may even be incorrect), as the option will not be shown in OSs that does not support it because of the `#if defined`.